### ISSUE TITLE
gridfs Problem after Update

### ISSUE THREAD

**Issue by lugaru90** [CollectionFS/Meteor-CollectionFS#588]
Hi,

when updating from Gridfs 0.0.27 to 0.0.30 i get the following error. Cant say where it starts:
```javascript
I20150305-14:21:44.160(1)? Exception in queued task: TypeError: Cannot read property 'start' of undefined
I20150305-14:21:44.160(1)?     at Object.FS.StorageAdapter.createReadStream (packages/cfs:gridfs/gridfs.server.js:78:1)
I20150305-14:21:44.160(1)?     at Object.self.adapter.createReadStreamForFileKey (packages/cfs:storage-adapter/storageAdapter.server.js:71:1)
I20150305-14:21:44.160(1)?     at Object.self.adapter.createReadStream (packages/cfs:storage-adapter/storageAdapter.server.js:79:1)
I20150305-14:21:44.160(1)?     at packages/cfs:tempstore/tempStore.js:364:1
I20150305-14:21:44.160(1)?     at runWithEnvironment (packages/meteor/dynamics_nodejs.js:108:1)
I20150305-14:21:44.160(1)?     at packages/meteor/dynamics_nodejs.js:121:1
I20150305-14:21:44.160(1)?     at CombinedStream._getNext (/home/lugaru/.meteor/packages/cfs_tempstore/.0.1.4.yo7l0f++os+web.browser+web.cordova/npm/node_modules/combined-stream/lib/combined_stream.js:83:3)
I20150305-14:21:44.160(1)?     at CombinedStream.resume (/home/lugaru/.meteor/packages/cfs_tempstore/.0.1.4.yo7l0f++os+web.browser+web.cordova/npm/node_modules/combined-stream/lib/combined_stream.js:132:10)
I20150305-14:21:44.160(1)?     at CombinedStream.pipe (/home/lugaru/.meteor/packages/cfs_tempstore/.0.1.4.yo7l0f++os+web.browser+web.cordova/npm/node_modules/combined-stream/lib/combined_stream.js:64:8)
I20150305-14:21:44.160(1)?     at saveCopy (packages/cfs:worker/fileWorker.js:176:1)
I20150305-14:21:44.161(1)? Exception in queued task: TypeError: Cannot read property 'start' of undefined
I20150305-14:21:44.161(1)?     at Object.FS.StorageAdapter.createReadStream (packages/cfs:gridfs/gridfs.server.js:78:1)
I20150305-14:21:44.161(1)?     at Object.self.adapter.createReadStreamForFileKey (packages/cfs:storage-adapter/storageAdapter.server.js:71:1)
I20150305-14:21:44.161(1)?     at Object.self.adapter.createReadStream (packages/cfs:storage-adapter/storageAdapter.server.js:79:1)
I20150305-14:21:44.161(1)?     at packages/cfs:tempstore/tempStore.js:364:1
I20150305-14:21:44.161(1)?     at runWithEnvironment (packages/meteor/dynamics_nodejs.js:108:1)
I20150305-14:21:44.161(1)?     at packages/meteor/dynamics_nodejs.js:121:1
I20150305-14:21:44.161(1)?     at CombinedStream._getNext (/home/lugaru/.meteor/packages/cfs_tempstore/.0.1.4.yo7l0f++os+web.browser+web.cordova/npm/node_modules/combined-stream/lib/combined_stream.js:83:3)
I20150305-14:21:44.161(1)?     at CombinedStream.resume (/home/lugaru/.meteor/packages/cfs_tempstore/.0.1.4.yo7l0f++os+web.browser+web.cordova/npm/node_modules/combined-stream/lib/combined_stream.js:132:10)
I20150305-14:21:44.161(1)?     at CombinedStream.pipe (/home/lugaru/.meteor/packages/cfs_tempstore/.0.1.4.yo7l0f++os+web.browser+web.cordova/npm/node_modules/combined-stream/lib/combined_stream.js:64:8)
I20150305-14:21:44.161(1)?     at saveCopy (packages/cfs:worker/fileWorker.js:176:1)
I20150305-14:21:44.161(1)? Exception in queued task: TypeError: Cannot read property 'start' of undefined
I20150305-14:21:44.161(1)?     at Object.FS.StorageAdapter.createReadStream (packages/cfs:gridfs/gridfs.server.js:78:1)
I20150305-14:21:44.161(1)?     at Object.self.adapter.createReadStreamForFileKey (packages/cfs:storage-adapter/storageAdapter.server.js:71:1)
I20150305-14:21:44.161(1)?     at Object.self.adapter.createReadStream (packages/cfs:storage-adapter/storageAdapter.server.js:79:1)
I20150305-14:21:44.161(1)?     at packages/cfs:tempstore/tempStore.js:364:1
I20150305-14:21:44.161(1)?     at runWithEnvironment (packages/meteor/dynamics_nodejs.js:108:1)
I20150305-14:21:44.162(1)?     at packages/meteor/dynamics_nodejs.js:121:1
I20150305-14:21:44.162(1)?     at CombinedStream._getNext (/home/lugaru/.meteor/packages/cfs_tempstore/.0.1.4.yo7l0f++os+web.browser+web.cordova/npm/node_modules/combined-stream/lib/combined_stream.js:83:3)
I20150305-14:21:44.162(1)?     at CombinedStream.resume (/home/lugaru/.meteor/packages/cfs_tempstore/.0.1.4.yo7l0f++os+web.browser+web.cordova/npm/node_modules/combined-stream/lib/combined_stream.js:132:10)
I20150305-14:21:44.162(1)?     at CombinedStream.pipe (/home/lugaru/.meteor/packages/cfs_tempstore/.0.1.4.yo7l0f++os+web.browser+web.cordova/npm/node_modules/combined-stream/lib/combined_stream.js:64:8)
I20150305-14:21:44.162(1)?     at saveCopy (packages/cfs:worker/fileWorker.js:176:1)
I20150305-14:21:44.162(1)? Exception in queued task: TypeError: Cannot read property 'start' of undefined
I20150305-14:21:44.162(1)?     at Object.FS.StorageAdapter.createReadStream (packages/cfs:gridfs/gridfs.server.js:78:1)
I20150305-14:21:44.162(1)?     at Object.self.adapter.createReadStreamForFileKey (packages/cfs:storage-adapter/storageAdapter.server.js:71:1)
I20150305-14:21:44.162(1)?     at Object.self.adapter.createReadStream (packages/cfs:storage-adapter/storageAdapter.server.js:79:1)
I20150305-14:21:44.162(1)?     at packages/cfs:tempstore/tempStore.js:364:1
I20150305-14:21:44.162(1)?     at runWithEnvironment (packages/meteor/dynamics_nodejs.js:108:1)
I20150305-14:21:44.162(1)?     at packages/meteor/dynamics_nodejs.js:121:1
I20150305-14:21:44.162(1)?     at CombinedStream._getNext (/home/lugaru/.meteor/packages/cfs_tempstore/.0.1.4.yo7l0f++os+web.browser+web.cordova/npm/node_modules/combined-stream/lib/combined_stream.js:83:3)
I20150305-14:21:44.162(1)?     at CombinedStream.resume (/home/lugaru/.meteor/packages/cfs_tempstore/.0.1.4.yo7l0f++os+web.browser+web.cordova/npm/node_modules/combined-stream/lib/combined_stream.js:132:10)
I20150305-14:21:44.162(1)?     at CombinedStream.pipe (/home/lugaru/.meteor/packages/cfs_tempstore/.0.1.4.yo7l0f++os+web.browser+web.cordova/npm/node_modules/combined-stream/lib/combined_stream.js:64:8)
I20150305-14:21:44.162(1)?     at saveCopy (packages/cfs:worker/fileWorker.js:176:1)
```

Installed CFS Packages Broken:
cfs:graphicsmagick           0.0.18  Adds `gm` to scope with the ability to perform GraphicsMagick or ImageMagick manipulation
cfs:gridfs                   0.0.30  GridFS storage adapter for CollectionFS
cfs:standard-packages        0.5.4  Filesystem for Meteor, collectionFS

Installed CFS Packages Works:
cfs:graphicsmagick           0.0.18  Adds `gm` to scope with the ability to perform GraphicsMagick or ImageMagick manipulation
cfs:gridfs                   0.0.27* GridFS storage adapter for CollectionFS
cfs:standard-packages        0.5.0* Filesystem for Meteor, collectionFS

image.js Collection
```javascript
var imageStore = new FS.Store.GridFS("images", {
  chunkSize: 1024*1024,
	transformWrite: function(fileObj, readStream, writeStream) {
	  // Transform the image into a 10x10px thumbnail
	  gm(readStream, fileObj.name()).resize('200', '200').stream().pipe(writeStream);
	}
});
//var thumbStore = new FS.Store.GridFS("thumbs", {
//	chunkSize: 1024*1024
//});


Images = new FS.Collection("images", {
  stores: [imageStore],
	filter: {
	  allow: {
	    contentTypes: ['image/*'] //allow only images in this FS.Collection
	  }
	}
});

Images.allow({
	insert: trueFunc,
	update: trueFunc,
	remove: trueFunc,
	download: trueFunc
});

function trueFunc() {return true;}
```

**Comment by obiwarn** [comment]
exactly the same error log here

**Comment by phsultan** [comment]
I experienced the same issue, and can confirm that latest modifications to cfs:gridfs in the 'devel' branch fixed it.

**Comment by aldeed** [comment]
Everything in devel branch should be released in 0.0.31, so I'll close this.
